sdk/maps/azure-maps-search/azure/maps/search/__init__.py

```diff
@@ -1,12 +1,50 @@
-# ------------------------------------
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-# ------------------------------------
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
 
 from ._version import VERSION
-from ._search_client import MapsSearchClient
+from ._enums import (
```

Comment:
Did you manually add these?
In future, these should all be moved into `_patch.py` so that this file does not get manually edited.